docs/audio.rst

```diff
@@ -38,6 +41,14 @@ Classes
 
     It takes just over 4 ms to play a single frame.
 
+    .. py:function:: copyfrom(other)
+
+        Overwrite the data in this ``AudioFrame`` with the data from another
+        ``AudioFrame`` instance.
+
+        :param other: ``AudioFrame`` instance to copy the data from.
```

Comment:
@microbit-mark do you have a suggestion on how to rephrase this in a way that it doesn't end with a preposition?

Reply:
Would this make more sense?

```suggestion
        :param other: ``AudioFrame`` instance from which to copy the data.
```

Reply:
Perfect, thanks!